nsdb-cluster/src/test/scala/io/radicalbit/nsdb/cluster/coordinator/AbstractReadCoordinatorSpec.scala

```diff
@@ -0,0 +1,208 @@
+/*
+ * Copyright 2018 Radicalbit S.r.l.
```

Comment:
Copyright needs to be updated?

Reply:
I'm not sure about that. @robbenti what's your thought about it?

Reply:
I'm not sure but probably we should add a date range like 

> Copyright 2017-2019 Radicalbit S.r.l.

Let's change this is a different PR